### DISCUSSION TITLE
BEISPIEL: Wie stelle ich eine Frage?

### DISCUSSION THREAD

**Discussion by mbosselmann** [neuefische/questions#11] [Guides & FAQ]
**Dies ist ein Beispiel Post, wie eine Frage gestellt werden soll und wie sie beantwortet wird und wie es aussieht, wenn eine Antwort als "Antwort" markiert wird.**

Hallo,
gerade arbeite ich an meiner Card Komponente und bekomme eine Fehlermeldung, die ich nicht verstehe. Sie bezieht sich anscheinend auf meine Styled Component und wird von ESLint erzeugt. 

Ich habe meinen aktuellen Stand committed und gepusht. [Dieser ist unter diesem Link erreichbar.](https://github.com/mbosselmann/capstone-template-test/commit/d28b15bfea07ab0ce4839739aadaa35843ee89ec) Der Branch heißt `card`.

**Beschreibung des Fehlers:**
**Wie kann man den Fehler reproduzieren?**
Direkt beim Starten des Development Servers wird diese Fehlermeldung im Browser angezeigt.
**Wo und wann tritt er auf?**
Direkt beim Starten des Development Servers.
**Was wird erwartet, was passiert?**
Ich erwarte, dass meine Komponente ohne Fehlermeldung rendert. Klicke ich die Fehlermeldung weg, wird diese trotzdem angezeigt, aber wenn ich den Development Server neustarte oder den Browser refreshe taucht die Fehlermeldung erneut auf.
**Was sind die Fehlermeldungen?**
```
ERROR

[eslint] 
src/components/Card/Card.js
  Line 18:17:  'styled' is not defined  no-undef

Search for the keywords to learn more about each error.
```
**Was habe ich bereits ausprobiert:**
Ich habe bereits nach diesem Fehler online gesucht und ich habe probiert die Styled Component neu zu schreiben, weil ich dachte, dass ich einen Fehler in der Syntax habe. Dennoch taucht dieser Fehler immer wieder auf.

Vielen Dank für eure Hilfe!

**Comment by PReichetanz** [comment]
Liebe Mareike,

der Fehler weist darauf hin, dass `styled` nicht importiert ist.
Versuche doch mal, den Import von styled zu setzen:

`import styled from 'styled-components`;

**Reply by mbosselmann** [reply]
Hallo Paul,
vielen Dank für deine Antwort! 
Das habe ich gemacht und der Fehler ist jetzt behoben. :)
Damit ist mein Problem gelöst. ✅

**Comment by mpagels** [comment]
#456 ist noch ein aktuelles Beispiel (Stand 01/2023) von @pixelass, was ebenfalls genutzt werden kann.
